### DISCUSSION TITLE
Can't upload file when exceeded 4-5MB

### DISCUSSION THREAD

**Discussion by MasterProvider** [vercel/next.js#47152] [Help]
### Summary

I can only upload a limit of 2MB but when I upload a file exceeded with 4-5MB it returns to error
it's working fine in localhost but when I deploy it in Vercel it returns to error.
//pages/api/upload.js
```
import { IncomingForm } from "formidable";

export const config = {
  api: {
    bodyParser: false,
  },
};

export default async function handler(req, res) {
  const data = await new Promise((resolve, reject) => {
    const form = new IncomingForm({ multiples: true });
    form.parse(req, (err, fields, files) => {
      if (err) return reject(err);
      resolve({ fields, files });
    });
  });
  console.log("Receiving");
  if (req.method === "PUT") {
    res.status(200).send(data.files);
  }
}

```

demo: https://upload-file-nextjs.vercel.app

### Additional information

```js
"axios": "1.3.4",
 "formidable": "2.1.1",
 "next": "13.2.4",
 "react": "18.2.0",
 "react-dom": "18.2.0"
```

### Example

https://codesandbox.io/p/sandbox/upload-file-nextjs-5uo9hw

**Comment by icyJoseph** [comment]
There's limit at 4.5MB:

> If a Serverless Function receives a payload in excess of the limit it will return an error → 413: FUNCTION_PAYLOAD_TOO_LARGE.

- https://vercel.com/docs/concepts/limits/overview
- https://vercel.com/docs/concepts/limits/fair-use-policy#general-limits
- https://vercel.com/docs/concepts/limits/overview#serverless-function-payload-size-limit
